core/src/value/person_name.rs

```diff
@@ -0,0 +1,326 @@
+use crate::dicom_value;
```

Comment:
If this module has to be public, then a small root-level piece of documentation is recommended. It should at least explain that this aims for an implementation of DICOM values with the VR PN (person name) as per [PS3.5 sect 6.2](https://dicom.nema.org/medical/dicom/current/output/chtml/part05/sect_6.2.html).